tracer/src/Datadog.Trace/NativeLoader.cs

```diff
@@ -0,0 +1,66 @@
+// <copyright file="NativeLoader.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace
+{
+    internal static class NativeLoader
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(NativeLoader));
+
+        private static bool IsAvailable
+        {
+            get
+            {
+                var fd = FrameworkDescription.Instance;
+                return fd.ProcessArchitecture != ProcessArchitecture.Arm && fd.ProcessArchitecture != ProcessArchitecture.Arm64;
```

Comment:
Just realised, this is also _currently_ false for the distribution NuGet, AAS, and when using custom tracing only. Hopefully we can fix those in the same release though!